### DISCUSSION TITLE
GET on groups

### DISCUSSION THREAD

**Discussion by collin-catalogic** [microsoftgraph/msgraph-sdk-go#89] [Q&A] [answered]
```
result, err := graphServiceClient.Groups().Get(nil)
```
This line of code does not return any groups, nor does it throw an error.
Trying this code referring from [docs](https://docs.microsoft.com/en-us/graph/api/group-list?view=graph-rest-1.0&tabs=go)

Tried the same, using the GRAPH API explorer:
URL: https://graph.microsoft.com/v1.0/groups
RESPONSE: 
```
{
    "error": {
        "code": "InternalServerError",
        "message": "Unable to find target address",
        "innerError": {
            "date": "2022-02-25T16:19:40",
            "request-id": "49c9bdcb-3878-464e-8f29-f440c3a72cad",
            "client-request-id": "cc87990b-783a-e8c1-1851-6a9745304078"
        }
    }
}
```
Is there an issue with the API or am i using it incorrectly?

**Comment by baywet** [comment, marked as answer]
are you signed in with a personal account or a professional one (AAD/Office 365/etc) ?

**Reply by collin-catalogic** [reply]
I am using my personal account for this one.

**Reply by baywet** [reply]
As you can see in the permissions tab, this endpoint doesn't support personal accounts https://docs.microsoft.com/en-us/graph/api/group-list?view=graph-rest-1.0&tabs=http#permissions
